app/src/main/java/com/nervousfish/nervousfish/data_objects/Contact.java

```diff
@@ -45,10 +66,42 @@ public Contact(final String name, final Collection<IKey> keys) {
         }
     }
 
+    /**
+     * Constructor for the {@link Contact} POJO for multiple {@link IKey}s, and
+     * the IBAN.
+     *
+     * @param name The name of the {@link Contact}
+     * @param keys The {@link Collection} of keys to initialize the {@link Contact} with
+     * @param iban  The IBAN of the {@link Contact}
+     */
+    public Contact(final String name, final Collection<IKey> keys, final IBAN iban) {
+        this.name = name;
+        for (final IKey key : keys) {
+            this.keys.add(SerializationUtils.clone(key));
+        }
+        this.iban = iban;
+    }
+
     public String getName() {
         return this.name;
     }
 
+    public IBAN getIban() {
+        return this.iban;
+    }
+
+    /**
+     * Retuns the string of the IBAN, and a dash if the object is null.
+     *
+     * @return String of the IBAN, and a dash if the object is null;
+     */
+    public String getIbanAsString() {
+        if (this.getIban() != null) {
```

Comment:
iban is property of this class itself, so a getter is not needed. Getters are only for classes from the World to access private variables they cannot access otherwise